sgv2-service-common/src/main/java/io/stargate/sgv2/common/grpc/DefaultStargateBridgeSchema.java

```diff
@@ -199,7 +216,15 @@ public void onCompleted() {
   }
 
   private void scheduleReconnect() {
-    executor.schedule(this::registerChangeObserver, getAndIncreaseDelayMs(), TimeUnit.MILLISECONDS);
+    try {
+      if (!executor.isShutdown()) {
+        executor.schedule(
+            this::registerChangeObserver, getAndIncreaseDelayMs(), TimeUnit.MILLISECONDS);
+      }
+    } catch (RejectedExecutionException e) {
+      // The code above can occasionally race, avoid printing a full stack trace
+      LOG.warn("Reconnection task was rejected, this is normal if the server is shutting down");
```

Comment:
Maybe include just the `getMessage`? Maybe that's not that useful though overall

Reply:
I would suggest adding `getMessage()` at first; if it proves to be useless can remove but maybe better not to assume it is.